src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length calendar--start-day">
```

Comment:
```suggestion
    <div class="calendar calendar--month-length-29 calendar--start-day-tue">
```
do not forget to specify values for modifiers